김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/data/Owner.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.mvvmstudy.data
+
+import com.google.gson.annotations.SerializedName
+
+data class Owner(
+    @SerializedName("login")
+    val login : String,
+    @SerializedName("avatar_url")
+    val avatar_Url : String
```

Comment:
변수명에는 보통 카멜케이스를 쓰는 것 같아요!
```suggestion
    val avatarUrl : String
```